src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int anInt = random.nextInt(5);
```

Comment:
The variable name 'anInt' is not descriptive. Use a name that reflects the purpose of the variable, such as 'figureType'.